content/documentation/staging/configuration/authentication/anonymous.adoc

```diff
@@ -0,0 +1,83 @@
+---
+title: "Anonymous strategy"
+date: 2020-08-25T00:00:00+02:00
+draft: false
+weight: 9
+---
+
+:toc: macro
+:toc-title: In this section:
+:keywords: authentication anonymous
+:icons: font
+:sectlinks:
+
+toc::[]
+
+== Introduction
+
+The `anonymous` strategy removes any authentication requirement. Users will
+have access to Kiali without providing any credentials.
+
+Although the `anonymous` strategy doesn't provide any access protection, it's
+valid for some use-cases. Some examples known from the community:
+
+* Exposing Kiali through a reverse proxy, where the reverse proxy is providing a custom authentication mechanism.
+* Exposing Kiali on an already limited network of trusted users.
+* When Kiali is accessed through `kubectl port-forward` or alike commands that allow usage of the cluster's RBAC capabilities to limit access.
+* When developing Kiali, where a developer has a private instance on his own machine.
+
+icon:bullhorn[size=1x]{nbsp} It's worth to empasize that the `anonymous`
+strategy will leave Kiali unsecured. If you are using this option, make sure
+that Kiali is available only to trusted users, or access is protected by other
+means.
+
+== Set-up
+
+To use the `anonymous` strategy, use the following configuration in the Kiali
+operator resource:
```

Comment:
We always call this resource the `Kiali CR`. Let's use this throughout to maintain consistency with all the other documentation we have.

```suggestion
CR:
```